lib/prometheus/client/data_stores/direct_file_store.rb

```diff
@@ -205,7 +244,7 @@ def aggregate_values(values)
               if @values_aggregation_mode == SUM
                 values.inject { |sum, element| sum + element }
               elsif @values_aggregation_mode == MAX
-                values.max
+                  values.max
```

Comment:
This was probably an accident, will resolve once we have an agreement this is a valid solution.